draft-terbu-sd-jwt-vc.md

```diff
@@ -62,72 +52,70 @@ the Subject, for example to prove eligibility to access certain services.
            |            |
            +------------+
                  |
-          Issues SD-JWT VC
-     and Issuer-Issued Disclosures
+      Issues Verifiable Credential
                  |
                  v
-           +------------+                                 +------------+
-           |            |                                 |   Status   |
-           |   Holder   |-------- optionally ------------>|  Provider  |
-           |            |       retrieves status          |            |
-           +------------+             VC                  +------------+
+           +------------+
+           |            |
+           |   Holder   |
+           |            |
+           +------------+
                  |
-         Presents SD-JWT VP
-   and Holder-Selected Disclosures
+    Presents Verifiable Credential
                  |
                  v
            +-------------+
-           |             |+
-           |  Verifiers  ||+
-           |             |||
-           +-------------+||
-            +-------------+|
+           |             |+                                  +------------+
+           |  Verifiers  ||+                                 |   Status   |
+           |             |||---------- optionally ---------->|  Provider  |
+           +-------------+||       retrieve credential       |            |
+            +-------------+|            status               +------------+
              +-------------+
 ~~~
-Figure: SD-JWT VC Issuance and SD-JWT VP Presentation Flow w/ Status
-
-In the three-party-model, Verifiers have to trust Issuers to make
-trustworthy statements about the Subject and they can additionally require that
-the Holder provides a proof that they are the intended Holder of the Verifiable
-Credential which can important for security reasons. This is only possible if
-an Issuer binds the Verifiable Credential to a specific Holder at the time of
-issuance. This process is referred to as Holder Binding and is further
+Figure: Three-Party-Model with optional Status Provider
+
+Verifiers can check the authenticity of the data in the Verifiable Credentials
+and optionally enforce Holder Binding, i.e., ask the Holder to prove that they
+are the intended holder of the Verifiable Credential by proving possession of a
+cryptographic key referenced in the credential. This process is further
```

Comment:
Should we limit the text to cryptographic holder binding?

Reply:
I made a suggestion below that cryptographic holder binding is "an example". Currently, it is limited to the `cnf` claim but the `cnf` claim has an extension mechanism. So, technically it is not limited from a data model perspective. IMO, this limitation can be done by specific profiles such as the hi-assurance profile.

Reply:
Agree, this is meant to cover cryptographic holder binding.